cli/index.ts

```diff
@@ -399,6 +406,7 @@ const builtYargs = createYargsCli({
           compiledGraph.tests
         );
         testResults.forEach(testResult => printTestResult(testResult));
+        return 0;
```

Comment:
this should only `return 0` if all tests were successful: https://source.cloud.google.com/tada-analytics/github_dataform-co_dataform/+/master:protos/core.proto;l=362?q=testresult

Reply:
👍